docs/visual-basic/language-reference/operators/mod-operator.md

```diff
@@ -14,27 +14,30 @@ helpviewer_keywords:
 ms.assetid: 6ff7e40e-cec8-4c77-bff6-8ddd2791c25b
 ---
 # Mod operator (Visual Basic)
-Divides two numbers and returns only the remainder.  
-  
-## Syntax  
-  
-```  
-number1 Mod number2  
-```  
+Divides two numbers and returns only the remainder.
+
+## Syntax
+
+```vb
+result = number1 Mod number2
+```
   
 ## Parts  
- `number1`  
- Required. Any numeric expression.  
-  
- `number2`  
- Required. Any numeric expression.  
-  
-## Supported types  
- All numeric types. This includes the unsigned and floating-point types and `Decimal`.  
-  
+ `result`
+ Required. Any numeric expression.
```

Comment:
```suggestion
 Required. Any numeric variable or property.
```
Any numeric expression isn't accurate. For example (3 + 5) = 16 Mod 4 generates a compiler error.

Reply:
Well, then other operator articles will need to have the same change too.